src/routes/changelog/entry/[entry]/+page.svelte

```diff
@@ -52,6 +86,37 @@
                                 </li>
                             </ul>
                             <h1 class="web-title web-u-color-text-primary">{data.title}</h1>
+                            <div class="share-post-section u-flex u-gap-16 u-margin-block-start-16 u-cross-center">
+                                <span
+                                    class="u-font-size-16 u-padding-inline-end-8"
+                                    style:font-weight="500">
+                                    SHARE
+                                </span>
+
+                                <ul class="u-flex u-gap-8">
+                                    {#each sharingOptions as sharingOption}
+                                        <li class="share-list-item">
+                                            <Tooltip placement="bottom" disableHoverableContent={true}>
+                                                <button
+                                                    class="web-icon-button"
+                                                    aria-label={sharingOption.label}
+                                                    on:click="{() => handleShareClick(sharingOption)}"
```

Comment:
lets do the same suggestion I had in the blog post 👍🏻

Reply:
Added separate tags for links and copy button.